src/api.jl

```diff
@@ -139,8 +139,8 @@ To include all rules, specify `filter_modules = nothing`.
 # Examples
 
 ```jldoctest
-julia> first(DiffRules.diffrules())
-(:Base, :log2, 1)
+julia> DiffRules.diffrules() |> collect |> sort |> first
+(:Base, :*, 2)
```

Comment:
Maybe we should just use more meaningful tests, e.g., something like

```suggestion
julia> first(DiffRules.diffrules()) isa Tuple{Symbol,Symbol,Int}
true

julia> (:Base, :log, 1) in DiffRules.diffrules()
true

julia> (:Base, :*, 2) in DiffRules.diffrules()
true
```
The `collect`, `sort` etc doesn't seem very nice in a documentation and the test is still not completely reliable as it could break if some other rule is added. (A bit unrelated, I'm also not a huge fan of the pipe operator 😛)

Reply:
Mm yes I agree. Looks nicer as you suggest. I'll do it.